hw-1/task1.js

```diff
@@ -0,0 +1,19 @@
+import * as readline from 'readline';
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.on('line', (text) => {
```

Comment:
`rl` variable name is too short. It can be not so obvious what this variable for in big projects

Reply:
Fixed.